--- init-core.sql.orig	2019-02-04 12:20:23.000000000 +0100
+++ init-core.sql	2019-02-04 12:27:34.143713373 +0100
@@ -151,7 +151,7 @@
 "user_auto_creation" bool NOT NULL,
 "dir_uri" varchar(255) NOT NULL,
 "dir_admin" varchar(255),
-"dir_password" varchar(255),
+"dir_password" text,
 "dir_connection_security" varchar(10),
 "dir_case_sensitive" bool NOT NULL,
 "dir_password_policy" bool NOT NULL,
