diff -Naur before/schema/init-core.sql after/schema/init-core.sql
--- before/schema/init-core.sql	2017-09-04 10:57:33.000000000 +0200
+++ after/schema/init-core.sql	2017-09-26 14:37:56.058610056 +0200
@@ -155,7 +155,7 @@
 "user_auto_creation" bool NOT NULL,
 "dir_uri" varchar(255) NOT NULL,
 "dir_admin" varchar(255),
-"dir_password" varchar(255),
+"dir_password" text,
 "dir_connection_security" varchar(10),
 "dir_case_sensitive" bool NOT NULL,
 "dir_password_policy" bool NOT NULL,
